scripts/gen_manifests.sh

```diff
@@ -50,6 +54,15 @@ SONATAFLOW_CR=manifests/01-sonataflow_${workflow_id}.yaml
 yq --inplace eval '.metadata.annotations["sonataflow.org/profile"] = "prod"' "${SONATAFLOW_CR}"
 
 yq --inplace ".spec.podTemplate.container.image=\"quay.io/orchestrator/serverless-workflow-${workflow_id}:latest\"" "${SONATAFLOW_CR}"
+yq --inplace ".spec.podTemplate.container.envFrom=[{\"secretRef\": { \"name\": \"${workflow_id}-creds\"}}]" "${SONATAFLOW_CR}"
+
+if test -f "secrets.properties"; then
+  ../kubectl create -n sonataflow-infra secret generic "${workflow_id}-creds" --from-env-file=secrets.properties --dry-run=client -oyaml > "manifests/00-secrets_${workflow_id}.yaml"
```

Comment:
kn-workflow cli starts counting from `01` for the resources it generates.
let's attempt to remain consistent with that.
also, pls change it to a single form: `manifests/01-secret_${workflow_id}.yaml`